### PR TITLE
Update AndroidManifest.xml from OpenGapps (com.google.android.tvlauncher)

### PR DESCRIPTION
**Pull request by roblav96**
After decompiling the `TVLauncher.apk` found in OpenGapps `open_gapps-arm64-9.0-tvmini-20210619.zip`, I found a few useful edits, also found pointers from the https://gitlab.com/etienn01/flauncher repo.

I've been testing all day on my devices, `Nvidia Shield TV 2019 Pro`, `ADT-3 Developer Kit`, and `Chromecast with Google TV`, no problems found. =]

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,16 +23,24 @@
 		android:icon="@drawable/ic_launcher"
 		android:label="@string/app_name"
 		android:supportsRtl="true"
+		android:usesCleartextTraffic="true"
```
Comment (nielsvanvelzen) [inline app/src/main/AndroidManifest.xml:26]: The app doesn't use networking (right now) and if it did it probably wouldn't allow any HTTP traffic, so this option is not needed.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -22,16 +23,24 @@
...
 		android:theme="@style/Theme.TVLauncher">
-		<activity android:name=".LauncherActivity">
-			<intent-filter>
+		<activity
+			android:name=".LauncherActivity"
+			android:stateNotNeeded="true"
+			android:excludeFromRecents="true"
+			android:launchMode="singleTask"
+			android:screenOrientation="landscape">
```
Comment (nielsvanvelzen) [inline app/src/main/AndroidManifest.xml:33]: Both landscape and portrait orientations work so we don't need to force landscape mode, so this option is not needed.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -22,16 +23,24 @@
...
+			<intent-filter android:priority="999">
```
Comment (nielsvanvelzen) [inline app/src/main/AndroidManifest.xml:34]: I don't think we need to set a different priority

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -4,6 +4,7 @@
 
 	<uses-permission android:name="android.permission.INTERNET" />
 	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+	<uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```
Comment (nielsvanvelzen) [inline app/src/main/AndroidManifest.xml:7]: The app does not use this permission right now so it shouldn't be added here.

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
Thanks for your pull request! I've left some notes (mainly options that are not needed).

**Review by nielsvanvelzen — APPROVED** [review]
Thank you!

**Comment by roblav96** [comment]
Cheers mate! 🍻

**Pull request merged**